HWs/HW2/Fibonacci.c

```diff
@@ -0,0 +1,54 @@
+#include <stdio.h>
+#include <stdbool.h>
+
+#define bound 47
+
+int iteratFibonacci(int number)
+{
+    int firstNumber = 0;
+    int secondNumber = 1;
+
+    for (int i = 0; i < number; ++i)
+    {
+        secondNumber = secondNumber + firstNumber;
+        firstNumber = secondNumber - firstNumber;
+    }
+
+    return firstNumber;
+}
+
+int recursFibonacci(int number)
+{
+    if (number == 0 || number == 1)
+    {
+        return number;
+    }
+
+    return recursFibonacci(number - 1) + recursFibonacci(number - 2);
+}
+
+void main()
+{
+    int number = 0;
+    int scan_res = 0;
+    bool flagNumber = true;
+
+    do
+    {
+        flagNumber = true;
```

Comment:
```suggestion
        isIncorrectInput = true;
```
Опять же неговорящее имя